src/MooLite/plugins/WhisperManager/DevConstants.ts

```diff
@@ -0,0 +1,565 @@
+import { ChatChannelTypeHrid } from "src/MooLite/core/chat/ChatChannelTypeHrid";
+import { ConversationMessage } from "src/MooLite/plugins/WhisperManager/ConversationMessage";
+
+export const contributorNames: string[] = ["LagunaE", "Sleutelkind"];
+export const betaTesterNames: string[] = ["JBQ", "knighterd", "Thessaloniki", "Houston"];
+export const helperNames: string[] = ["Champ", "ML", "Maxion", "hoador"];
```

Comment:
I actually don't like some people having a special name.

Cheze is working on a chat icon for community contributors, and I'm sure the relevant people will be accredited that way. It's not really up to us

Reply:
I think you misunderstood. This is for people who helped, tested, or contributed to this plugin. It's a small thanks to them :D I'm happy to remove you from the list if you like.

Reply:
I understand, and I don't think that is something we should highlight in the plugin itself.

I'm all for adding an "author/maintainer" attribute to each plugin, but not change the behaviour of plugins based on who we believe deserve recognition

Reply:
This doesn't change the behavior at all. It simply determines the color of their name on the tabs. It's fairly common for beta testers and the like to be distinguished somehow in the final product. Look at MWI itself with the beta bunnies.

I disagree. The people listed here were instrumental in helping me pump this out so fast. I spoke with them about the colors they like, fiddled with it to get the contrast right, and checked that they like it.

Can we just restrict this feature to it's current implementation instead of removing it? I don't have to use these anywhere else, but I think this use case is fine.